pyface/ui/qt/fields/toggle_field.py

```diff
@@ -60,6 +60,16 @@ def _observe_control_value(self, remove=False):
         else:
             self.control.toggled.connect(self._update_value)
 
+    def _get_control_alignment(self):
+        """ Toolkit specific method to get the control's alignment. """
+        # dummy implementation
+        return self.alignment
+
+    def _set_control_alignment(self, alignment):
+        """ Toolkit specific method to set the control's alignment. """
+        # use stylesheet for button alignment
+        self.control.setStyleSheet(f"text-align: {alignment}")
```

Comment:
Would this remove any previous style-sheet modifications that are unrelated to alignment?

Reply:
Yes, for widget-level style-sheet modifications but not application-wide style-sheets, I think.  I'm OK with this because Pyface doesn't expose the style-sheets anywhere else and anyone who is dropping down to the Qt level for a Field should be aware that they may be stepping on the toes of the field infrastructure.

If we find that we need to set other characteristics via style sheets we may need to add a layer to manage merging stylesheet settings, but that would likely be at the level of the Widget.  So for a future PR if it is an issue